libr/anal/p/anal_sh.c

```diff
@@ -761,10 +1009,10 @@ static int (*first_nibble_decode[])(RAnal*,RAnalOp*,ut16) = {
  * routines defined in first_nibble_decode table
  */
 static int sh_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int len) {
-	ut8 op_MSB,op_LSB;
+	ut8 op_MSB, op_LSB;
 	int ret;
 	if (!data || len < 2) {
-		return 0;
+			return 0;
```

Comment:
wrong indent

Reply:
I removed function, but i'm just starting to test that. There still might be dozens of bugs